rfc-006-supported-software.md

```diff
@@ -0,0 +1,17 @@
+# Involve the Support Team in exotic software choices
+
+## Summary
+
+The support team must be involved in the decision making where major technology choices increase the burden on the support team.
+
+## Problem
+
+Currently the support team are expected to provide both BAU support, and offer improvements to any platforms that delivery teams ship.
```

Comment:
As we have a vast number of applications and infrastructure spread across multiple customers, each time that they differ drastically, it means the knowledge required to switch to and work on any of these increases substantially. Building up this knowledge to anyone joining support, or when a new project is onboarded into support, it takes time. It means it's slower to onboard new people and projects into support which affects profits and the service we provide. 

Also, building custom solutions/or rolling our own solutions for things that already exist, or closely enough would fit a requirement, I feel those should be used instead, as this also reduces the amount of moving parts that we need to support.

I feel that testing these new technologies on internal projects might be a good test bed for them, and also gives a bit of time to come to a consensus on best practices that we should then adopt in our customer projects.